### PR TITLE
#333 - clean-up ports in development environment

### LINKED ISSUE #333: Clean-up ports in development environment
Today I was setting up the project for 10 minutes, because all ports for all services are default ones and are conflicting with other projects. 

This should be cleaned up accordingly to https://docs.google.com/spreadsheets/d/1hYOqef3mQ1vPPuYHlM5j45HeIx1IsaMrZ9gq-ow_BkQ/edit#gid=0

### PR DESCRIPTION
**Pull request by EwelinaSkrzypacz**
This should close #333.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -37,7 +37,7 @@ MAIL_FROM_NAME="${APP_NAME}"
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 GOOGLE_CALENDAR_ENABLED=true
-GOOGLE_REDIRECT=http://localhost/login/google/end
+GOOGLE_REDIRECT=http://localhost:8751/login/google/end
```
Comment (krzysztofrewak) [inline .env.example:40]: Why not `toby.blumilk.localhost`?

File: docker-compose.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ services:
       - ./environment/dev/app/supervisord.conf:/etc/supervisor/custom-supervisord.conf:ro
       - .:/application
     ports:
-      - ${DOCKER_APP_HOST_PORT:-8181}:80
+      - ${DOCKER_APP_HOST_PORT:-80}:80
```
Comment (krzysztofrewak) [inline docker-compose.yml:32]:
```suggestion
      - ${DOCKER_APP_HOST_PORT:-8751}:80
```

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (EwelinaSkrzypacz) [reply .env.example:40]: Because login by Google doesn't work with `toby.blumilk.localhost`, but I can change this.

Comment (krzysztofrewak) [reply .env.example:40]: That would be nice.

**Review by krzysztofrewak — APPROVED** [review]

**Review by Baakoma — APPROVED** [review]

**Pull request merged**